### ISSUE TITLE
protein vs peptides

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12259]
In GO, do we distinguish between a protein and a peptide. I thought that at some point we decided that we didn't?
However, I am looking for protease inhibitor. I can find:

peptidase inhibitor activity
The definition fits:
Stops, prevents or reduces the activity of a peptidase, any enzyme that catalyzes the hydrolysis peptide bonds.

but, it has related synonym "protease inhibitor activity"
Should this be exact?

**Comment by ValWood** [comment]
inconsistent anyway:
http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008233
peptidase activity exact synonym protease activity

We do however have lots of 'peptide' terms in other contexts. Maybe we only discussed this previously in the context of proteolysis...

**Comment by paolaroncaglia** [comment]
I do have memories of previous discussions, but let me make sure with other editors if I remember correctly where we're at.

**Comment by paolaroncaglia** [comment]
Discussed at editors call yesterday. Relevant bits:
"Harold: protein implies it is coded. Peptides could be pieces of protein coming from anywhere. What is difference between peptidase and protease activity? If only talking about peptide bonds none. Unless we want to consider substrate specificity [which we don't - I think it was resolved that these were outside the scope of GO for these enzymes]. Decision: have peptidase activity, and make protease a narrow synonym. Synonyms are used to help find the right term. Peptidase reflects the general peptide bond hydrolase activity, and in some cases they are also proteins."
(For reference, full minutes here: http://wiki.geneontology.org/index.php/Ontology_meeting_2016-02-04)

So I'll look at terms that have 'peptidase' and/or 'protease' as strings in their names and/or synonyms, and I'll edit them for consistency.

Note for future reference if ever needed: I believe the above is an appropriate fix for GO, but should we need (and have time) to examine this in more detail, Melanie suggested this paper http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1147080/?page=1
and Paul T reminded us that Neil Rawlings at EBI (MEROPS) might be a precious source of information.

**Comment by paolaroncaglia** [comment]
I looked at terms whose names contain ‘peptidase’. If they have ‘protease’ synonyms, I made sure the synonym scope is narrow.
Note: For the time being, I only modified scopes of ‘peptidase’ synonyms that were already present; I did not add new synonyms. If existing synonyms have ‘related’ scope, I left them as they are.

Note for self: refined criteria I used for searching (in OBO-Edit):

find terms that have a name that contains the value peptidase + find terms that have an ID that starts with the value GO: + find terms that don’t have a Is_obsolete
=> 135 matches

Only a few terms remain on which I didn’t work, these are TG terms with a ton of synonyms - most likely, they can be ignored without any danger:

negative regulation of aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process
negative regulation of metalloendopeptidase activity involved in amyloid precursor protein catabolic process
negative regulation of peptidase activity in other organism involved in symbiotic interaction
positive regulation of aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process
positive regulation of metalloendopeptidase activity involved in amyloid precursor protein catabolic process
regulation of aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process
regulation of metalloendopeptidase activity involved in amyloid precursor protein catabolic process
